chlorophyll/codeview.py

```diff
@@ -108,6 +100,29 @@ def __init__(
         self._set_lexer(lexer)
         self._set_color_scheme(color_scheme)
 
+    @property
+    def context_menu(self) -> Menu:
+        if self._context_menu is None:
+            self._context_menu = self._create_context_menu()
+            super().bind("<Button-3>", lambda e:self._context_menu.tk_popup(e.x_root, e.y_root))
```

Comment:
Nit:
```suggestion
            super().bind("<Button-3>", lambda e: self._context_menu.tk_popup(e.x_root, e.y_root))
```

Also, this doesn't work on macOS, as it doesn't have Button-3, it needs to be something like `Control-Button-1`, but I don't know from the top of my head.

Reply:
I was wondering that myself but thought it did lol

Reply:
What are we to do of Button-3?
I now read somewhere "1=left click, 2=right click & 3=middle click" in tkinter. And AI is also vomiting some unreasonable shit. Maybe need to read "official docs"

Reply:
For me it's always just faster tu pull up TkDocs: https://tkdocs.com/tutorial/menus.html

Reply:
Reading the docs, I believe we're accepting this?
```python
if tk.Tk().tk.call("tk", "windowingsystem") == "aqua":
    super().bind("<Control-Button-1>", lambda e: self._context_menu.tk_popup(e.x_root + 10, e.y_root + 10))
else:
    super().bind("<Button-3>", lambda e: self._context_menu.tk_popup(e.x_root + 10, e.y_root + 10)) 
```
Confirm, then I'll commit the final thing myself.

Reply:
I already committed :)

Reply:
That's great... (I think GitHub Mobile should notify if such a thing occurs)
So now we can say it's resolved?